client/flutter/lib/pages/home_page.dart

```diff
@@ -31,16 +28,6 @@ class _HomePageState extends State<HomePage> {
   final FirebaseAnalytics analytics;
   _HomePageState(this.analytics);
 
-  @override
-  void initState() {
```

Comment:
This widget (`HomePage`) could, and should IMHO, now be a `StatelessWidget`

Reply:
Agreed, I don't want to create merge conflicts though.

Reply:
I don't think there will be any 🤔 , because the `build` method of the `_HomePageState` will just move up in the file/changes to the stateless `HomePage`. So possibly only on line 19.

Reply:
Done